### ISSUE TITLE
Constant bugs at random intervals that block use of action bar

### ISSUE THREAD

**Issue by Rezy942** [funkydude/BasicChatMods#82]
Not much hope as i see author isnt active here but i will leave this here:

![image](https://user-images.githubusercontent.com/122007087/213870863-3fd74999-c15c-4cc6-8d80-bc8a56e76b1a.png)

2x [ADDON_ACTION_FORBIDDEN] AddOn 'BasicChatMods' tried to call the protected function 'UseAction()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `UseAction'
[string "@FrameXML/SecureTemplates.lua"]:364: in function `handler'
[string "@FrameXML/SecureTemplates.lua"]:690: in function <FrameXML/SecureTemplates.lua:672>
[string "@FrameXML/SecureTemplates.lua"]:704: in function <FrameXML/SecureTemplates.lua:697>
[string "@FrameXML/SecureTemplates.lua"]:746: in function `SecureActionButton_OnClick'
[string "@FrameXML/ActionButton.lua"]:109: in function `TryUseActionButton'
[string "@FrameXML/ActionButton.lua"]:144: in function `ActionButtonDown'
[string "ACTIONBUTTON2"]:2: in function <[string "ACTIONBUTTON2"]:1>

Happens randomly, cant pinpoint. Blocks action bars. Forced to reload. Considering deleting until fixed sadly.
Perhaps could be when clicking on an item link in chat. Idk.

**Comment by funkydude** [comment]
> Not much hope as i see author isnt active here

What is this even? Starting a request for help with a rant based on your feature requests not being fulfilled within 2 weeks?

When you have actual steps to reproduce this issue, feel free to let me know. You're the first person to report such an issue.

**Comment by Rezy942** [comment]
Rant? Kind of a stretch dont you think?
I saw half of the posts on here go unanswered so I came to a conclusion that you dont bother to reply.
And dont bother. Youre telling me to reproduce a typical taint issue caused by blizzard UI that many addons are having problems with right now. Going to go ahead and close it, as you seem to have been too busy bitching rather than wondering why your addon does more harm than good.

Btw a note - 99% of users dont go and post bug reports. They either ignore and move on or straight up delete.
Dont mistake lack of interest in ur addon with "uR the FirsT THoo"
